src/services/smartWallet.js

```diff
@@ -299,7 +299,7 @@ class SmartWallet {
       });
   }
 
-  getAccountStakedAmount(tokenAddress: ?string): BigNumber {
+  getAccountStakedAmount(tokenAddress: ?string): Promise<BigNumber> | BigNumber {
```

Comment:
Isn't `Promise<BigNumber>` enough?

Reply:
No, as next line there is an early return ` if (!tokenAddress) return new BigNumber(0);` not wrapped in promise.